sec_certs/analyze_certificates.py

```diff
@@ -368,9 +373,15 @@ def build_cert_references(filter_rules_group, all_items_found):
 
 
 def analyze_references_graph(filter_rules_group, all_items_found, filter_label):
```

Comment:
I thought that the FIPS code no longer relies on the older CC code. But now I see that it indeed does and it mixes with the CC code in a really weird way, where functions are used for both FIPS and CC and its really just an interwoven mess. This needs to be address, at some point we want to drop the old CC code right?

Reply:
@petrs suggested this - he said there was no reason to have the same functionality implemented multiple times. We can probably discuss it together.